src/memory/server/MemoryServer.ts

```diff
@@ -55,20 +55,20 @@ export class MemoryServer {
                  <head>
                      <meta charset="utf-8">
                      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-                     <link href="${newPanel.webview.asWebviewUri(
-                         vscode.Uri.joinPath(
-                             context.extensionUri,
-                             'dist/MemoryBrowser.css'
-                         )
-                     )}" type="text/css" rel="stylesheet"></link>
+<link href="${newPanel.webview.asWebviewUri(
+            vscode.Uri.joinPath(
+                context.extensionUri,
+                'dist/MemoryBrowser-bundle.css'
+            )
+        )}" type="text/css" rel="stylesheet"></link>
                  </head>
                  <body>
                      <div id="app"></div>
                      <script src="${newPanel.webview
                          .asWebviewUri(
                              vscode.Uri.joinPath(
                                  context.extensionUri,
-                                 'dist/MemoryBrowser.js'
+                                 'dist/MemoryBrowser-bundle.js'
```

Comment:
Same here: We need to avoid changing the name and the location of the css and js files as possible as we can, it is breaking down the components who are referencing cdt-gdb-vscode.